dev/com.ibm.ws.concurrency.policy/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,45 @@
+maxPolicy.loose.desc=Maximum concurrency is loosely enforced, in that tasks are allowed to run on the task submitter's thread without counting against maximum concurrency.
+maxPolicy.strict.desc=Maximum concurrency is strictly enforced, in that tasks that run on the task submitter's thread count towards maximum concurrency. This policy does not allow running on the task submitter's thread when already at maximum concurrency.
+
+maxQueueSize=Maximum queue size
+maxQueueSize.desc=Specifies the maximum number of tasks that can be queued for execution at any given point in time. As tasks are started, canceled, or aborted, they are removed from the queue. When the queue is at capacity and another task is submitted, the behavior is determined by the maximum wait for enqueue and run-if-queue-full attributes. To ensure that a specific number of tasks can be queued within a short interval of time, use a maximum queue size that is at least as large as that amount. The default maximum queue size is unbounded, up to Integer.MAX_VALUE. Maximum queue size can be updated while tasks are in progress and/or queued for execution. If the maximum queue size is reduced below the current number of queued tasks, the update goes into effect gradually, as queued tasks execute naturally or are canceled by the user, rather than automatically canceling the excess queued tasks.
+
+maxWaitForEnqueue=Maximum wait for enqueue
+maxWaitForEnqueue.desc=Specifies the maximum duration of time to wait for enqueuing a task. If unable to enqueue the task within this interval, the task submission is subject to the run-if-queue-full policy. When the maximum wait for enqueue is updated, the update applies only to tasks submitted after that point. Tasks submissions that were already waiting for a queue position continue to wait per the previously configured value.
+
+runIfQueueFull=Run if queue full
+runIfQueueFull.desc=Applies when using the execute or submit methods. Indicates whether or not to run the task on the submitter's thread when the queue is full and the maximum wait for enqueue has been exceeded. When configured to false, the task submission is rejected after the maximum wait for enqueue elapses instead of running on the submitter's thread.
+
+startTimeout=Start timeout
+startTimeout.desc=Specifies the maximum amount of time that may elapse between task submission and task starting. By default, tasks do not time out. If both a maximum wait for enqueue and a start timeout are enabled, the start timeout should be configured larger than the maximum wait for enqueue such that remains possible to start tasks after they have waited for a queue position. When the start timeout is updated while in use, the new start timeout value applies to tasks submitted after the update occurs.
```

Comment:
+startTimeout.desc=Specifies the maximum amount of time that may elapse between the task submission and the task start. By default, tasks do not time out. If both a maximum wait for enqueue and a start timeout are enabled, configure the start timeout to be larger than the maximum wait for enqueue. This start timeout configuration makes it possible to start tasks after they wait for a queue position. When the start timeout is updated while in use, the new start timeout value applies to tasks submitted after the update occurs.

Reply:
This proposed update, "This start timeout configuration makes it possible to start tasks after they wait for a queue position," is untrue.  Lacking the start timeout, tasks can always start. With the start timeout, start is rejected when the timeout is exceeded.  I will agree to make the updates as requested after having removed this sentence.